CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
+## 2018-06-13 - Version 4.3.2
+
+### Summary
+
+This is a minor release with bug fixes.
+
+### Fixed
+- iis_website, with a port binding of 443, does not start. ([MODULES-7173](https://tickets.puppetlabs.com/browse/MODULES-7173))
```

Comment:
iis_website should be preformatted in backticks.

There should be a one-line break between header and body copy.